src/js/utils/specs/string-utils-spec.js

```diff
@@ -0,0 +1,58 @@
+define(
+    [
+        '../../utils/spec-helpers/spec-helpers.js',
+        '../string-utils.js'
+    ],
+    function(SpecHelpers, StringUtils) {
+        'use strict';
+
+        describe('StringUtils', function() {
+            describe('interpolateEscape', function() {
+                it('can interpolate a string with no parameters provided', function() {
+                    expect(StringUtils.interpolateEscape('Hello, world').toString()).toBe(
```

Comment:
It is subtly different because it tests the case where the options parameter isn't even provided. I tried to think of a way to incorporate this into the other tests but it seemed simpler to keep it separate.